content/docs/user-guide/experiment-management/sharing-experiments.md

```diff
@@ -0,0 +1,202 @@
+referenced in `dvc.lock` file.
+
+[run-cache]: /doc/user-guide/project-structure/internal-files#run-cache
+
+## Listing experiments remotely
+
+In order to list experiments in a DVC project, you can use the `dvc exp list`
+command. With no command line options, it lists the experiments in the current
+project.
+
+You can supply a Git remote name to list the experiments:
+
+```dvc
+$ dvc exp list origin
+main:
+	cnn-128
+	cnn-32
+	cnn-64
+	cnn-96
+```
+
+Note that by default this only lists experiments derived from the current commit
+(local `HEAD` or default remote branch). You can list all the experiments
+(derived from from every branch and commit) with the `--all` option:
+
+```dvc
+$ dvc exp list origin --all
+0b5bedd:
+	exp-9edbe
+0f73830:
+	exp-280e9
+	exp-4cd96
+	...
+main:
+	cnn-128
+	...
+```
+
+When you don't need to see the parent commits, you can list experiment names
+only, with `--names-only`:
+
+```dvc
+$ dvc exp list origin --names-only
+cnn-128
+cnn-32
+cnn-64
+cnn-96
+```
+
+## Downloading experiments from remotes
+
+When you clone a DVC repository, it doesn't fetch any experiments by default. In
+order to get them, use `dvc exp pull` (with the Git remote and the experiment
+name), for example:
+
+```dvc
+$ dvc exp pull origin cnn-64
+```
+
+This pulls all the necessary files from both remotes. Again, you need to have
+both of these configured (see this
+[earlier section](#prepare-remotes-to-share-experiments)).
+
+You can specify a remote to pull from with `--remote` (`-r`).
+
+DVC can use multiple threads to download files (4 per CPU core typically). You
+can set the number with `--jobs` (`-j`).
+
+If an experiment being pulled already exists in the local project, DVC won't
+overwrite it unless you supply `--force`.
+
+### Pulling all experiments
+
+You can create a loop to pull all experiments from `origin` (Git remote) like
+this:
+
+```dvc
+$ dvc exp list --all --names-only | while read -r expname ; do \
+    dvc exp pull origin ${expname} \
+done
+```
+
+## Example: Creating a directory for an experiment
+
+A good way to isolate experiments is to create a separate home directory for
+each one.
+
+> Another alternative is to use `dvc exp apply` and `dvc exp branch`, but here
+> we'll see how to use `dvc exp pull` to copy an experiment.
+
+Suppose there is a <abbr>DVC repository</abbr> in `~/my-project` with multiple
+experiments. Let's create a copy of experiment `exp-abc12` from there.
+
+First, clone the repo into another directory:
+
+```dvc
+$ git clone ~/my-project ~/my-experiment
+$ cd ~/my-experiment
+```
+
+Git sets the `origin` remote of the cloned repo to `~/my-project`, so you can
+see your all experiments from `~/my-experiment` like this:
+
+```dvc
+$ dvc exp list origin
+main:
+	exp-abc12
+	...
+```
+
+If there is no DVC remote in the original repository, you can define its
+<abbr>cache</abbr> as the clone's `dvc remote`:
+
+```dvc
+$ dvc remote add --local --default storage ~/my-project/.dvc/cache
+```
```

Comment:
Should we add a warning not to `git commit/push` the changes to `origin`? I'm not even sure what side-effects that could cause.

Reply:
`--local` config is `.gitignore`d I believe, right?

Reply:
True! I added a note about that, just to be sure the reader knows the importance of `--local` here.